bears/python/PyImportSortBear.py

```diff
@@ -182,7 +184,18 @@ def run(self, filename, file,
 
         if new_file != tuple(file):
             diff = Diff.from_string_arrays(file, new_file)
+            all_imports = ''
+            for line_nr in sorted(diff._changes):
+                linediff = diff._file[max(line_nr - 1, 0)]
+                list_of_imports = re.split(', +',
+                                           linediff.rstrip())
```

Comment:
fits into a single line (improves readability because of the comma inside the quotes, that's kinda confusing :3)